docs/source/getting_started/welcome_tour.rst

```diff
@@ -123,15 +123,14 @@ Putting all the pieces together, our trainer looks something like this:
 
 .. code-block:: python
 
-    from composer.core.time import Time
-    from composer.core.types import DataLoader
-
-    model = SomeComposerModel() # a Composer model which subclasses `ComposerModel`
-    max_duration = Time(10, TimeUnit.EPOCH) # max duration to train for
-    rank_zero_seed = 0
-    dataloader = (...) # load some dataset. Initialize the same way you would `torch.utils.DataLoader`.
-
-    state = State(model, max_duration, rank_zero_seed, dataloader)
+    from composer import Time
+
+    state = State(
+        model=your_model,  # ComposerModel 
```

Comment:
Are these valid kwargs? In the docs [here](https://docs.mosaicml.com/en/v0.5.0/api_reference/composer.core.state.html#composer.core.state.State) they are not listed as kwargs.